src/pages/css/properties/text-decoration-property/index.md

```diff
@@ -1,15 +0,0 @@
----
-title: Text Decoration Property
```

Comment:
Instead of removing the stub from the `properties/` folder, can you move the existing article into the properties folder?